src/test/java/co/rsk/federate/BtcToRskClientTest.java

```diff
@@ -102,6 +97,46 @@ void setup() throws PeginInstructionsException, IOException {
         btcToRskClientBuilder = new BtcToRskClientBuilder();
     }
 
+    @Test
+    void start_withExistingFederationMember_doesntThrowError() throws Exception {
+        BitcoinWrapper bw = new SimpleBitcoinWrapper();
+        SimpleFederatorSupport fh = new SimpleFederatorSupport();
+        BtcECKey btcPublicKeyFromMember = BtcECKey.fromPublicOnly(Hex.decode("02cd53fc53a07f211641a677d250f6de99caf620e8e77071e811a28b3bcddf0be1"));
+        FederationMember fedMember = FederationMember.getFederationMemberFromKey(btcPublicKeyFromMember);
+        fh.setMember(fedMember);
+        BtcToRskClient client = createClientWithMocks(bw, fh);
+        assertDoesNotThrow(() -> client.start(genesisFederation));
+    }
+
+    @Test
+    void start_withNoFederationMember_doesntThrowError() throws Exception {
+        BitcoinWrapper bw = new SimpleBitcoinWrapper();
+        SimpleFederatorSupport fh = new SimpleFederatorSupport();
+
+        BtcECKey btcPublicKeyFromFakeMember = BtcECKey.fromPublicOnly(Hex.decode("02cd53fc53a07f211641a677d250f6de99caf620e8e77071e811a28b3bcddf0be0"));
```

Comment:
```suggestion
        BtcECKey btcPublicKeyFromFakeMember = BtcECKey.fromPublicOnly(Hex.decode("00000000000000....."));
```
To make it clear that it not part of the federation